src/main/java/com/sprints/validations/SprintsValidations.java

```diff
@@ -21,4 +25,12 @@ public void sprintValidateInBacklog(Sprint sprint){
 			}
 		}
 	}
+	
+	public void sprintsEndDateValidations(SprintDomain sprintDomain) {
+		if(sprintDomain.getEnd_date().isBefore(LocalDate.now()) || sprintDomain.getEnd_date().isEqual(LocalDate.now())) {
+			if(sprintDomain.getActive() == true) {
```

Comment:
My understanding in this is that we're going to validate if the sprint is active or not after validating the endDate of the sprint? Is that right? or do we need to validate if the sprint is active first?

Reply:
Yes, we first check the date, then if the sprint is active or not

Reply:
Why do we need to validate first the date, and after if the sprint is active or not? why not inverted?

Reply:
In this case since this is a end-date validations specifically, we first validate the date, in case we need to make another validation according to the date, it would be easier this way, it is more like a scalability thing,just in case we need to add more code according to the date

Reply:
I'm not sure of this, do we need to validate the EndDate for no active Sprints? If not please reflect that change.